Sources/Views/ALKInformationCell.swift

```diff
@@ -171,16 +174,21 @@ final class ALKInformationCell: UITableViewCell, Localizable {
     }
 
     fileprivate func setupConstraints() {
+        contentView.subviews.forEach({ $0.removeFromSuperview() })
```

Comment:
See if there's a better way to refresh than removing each view. It will be costly to do these updates everytime.

Reply:
instead of removing & adding again, added show/hide logic for those lines